components/02-components/tables/01-simple-table/README.md

```diff
@@ -1 +1,9 @@
-*There are no notes for this component yet.* 
+Simple tables are the standard table styling component. They create a bold row header and basic text entries.
+
+The `display` modifier class adds vertical rules, alternating background colors, and more padding between the cell content and the top/bottom cell borders. Simple tables with display modifiers are visually similar to `.datatables` and are most commonly used for legal resources search results lists, which present legal case results in a format that’s visually consistent with filtered data results.
```

Comment:
Should this be `.display` to be the same as the reference to `.datatables`?

Reply:
I wasn't really sure what the best way to write about it was, since if you want to add display styles, you'd use `.simple-table--display`. So would it make more sense to say `.display` or `--display` ?

Reply:
I would put the name of the entire class, so in this case it would be `.simple-table--display` to avoid any confusion.

Reply:
👍 will do, thanks!